### ISSUE TITLE
War missing web.xml in WEB_INF?

### ISSUE THREAD

**Issue by KnowledgeGarden** [GateNLP/mimir#6]
Just booted the latest war file in a fresh tomcat 8+ on 64-bit ubuntu. Tomcat cannot find it. I looked inside; while all the other apps in tomcat had a web.xml, I found none in mimir. Perhaps I'm missing something?

**Comment by ianroberts** [comment]
It isn’t supposed to have a `web.xml` - that file is optional in the version of the servlet specification used by Grails 3 (the framework underpinning Mimir 6+). What exactly is the error you get from Tomcat?

**Comment by KnowledgeGarden** [comment]
404 not found. Tomcat opened the war, using the full name of that directory failed to boot it. The war was dropped in tomcat's webapps directory

**Comment by ianroberts** [comment]
So you dropped `mimir-cloud-6.1.war` into the Tomcat webapps folder, but `http://localhost:8080/mimir-cloud-6.1/` gives a 404?

What (if any) errors do you see in the Tomcat log files?

**Comment by KnowledgeGarden** [comment]
Catalina log says springboot is missing a dependency. I'm going to attempt to attach that log here.
[catalina.2019-06-21.log](https://github.com/GateNLP/mimir/files/3315704/catalina.2019-06-21.log)

**Comment by KnowledgeGarden** [comment]
other factors: maven 3.6 and openjdk-11 are installed

**Comment by greenwoodma** [comment]
Right, that problem is because you are using jdk 11. We're not sure why it's a problem but a fix that worked in java 9 and 10v had stopped working in 11. I've fixed it in the current master of gate-core but not updated Mimir to pull that in yet.

If you can run on Java 10 the error should go away.

**Comment by ianroberts** [comment]
This is probably related to https://github.com/GateNLP/gate-core/issues/83 which was fixed a couple of days ago - it seems JAXB is no longer available by default in Java 11. Mimir 6.1 _will_ work if you run tomcat under Java 8 instead of 11 (this is what our docker image does), and I’ll mark this issue as fixed once the latest snapshot of Mimir is updated to depend on `gate-core` 8.7-SNAPSHOT, since that includes the JAXB dependency required by Java 11.

**Comment by KnowledgeGarden** [comment]
I'm not in a hurry to use it; it's going to become important in a few months, so I'll wait. I'm assuming this will self-resolve soon, with another release. Meanwhile, I ran into similar problems with maven projects on that nix box; the fix was adding this to the pom's `<build>` section:

            `
			<plugin>
			   <groupId>org.apache.maven.plugins</groupId>
			   <artifactId>maven-compiler-plugin</artifactId>
			   <configuration>
			       <source>1.6</source>
			       <target>1.6</target>
			   </configuration>
			</plugin>`
